client/my-sites/backup/backup-contents-page/index.tsx

```diff
@@ -58,7 +58,32 @@ const BackupContentsPage: FunctionComponent< OwnProps > = ( { rewindId, siteId }
 					<div className="backup-contents-page__header daily-backup-status status-card">
 						<div className="status-card__message-head">
 							<img src={ cloudIcon } alt="" role="presentation" />
-							<div className="status-card__title">{ translate( 'Backup contents from:' ) }</div>
+							<div className="status-card__header-content">
+								<div className="status-card__title">{ translate( 'Backup contents from:' ) }</div>
+								<div className="status-card__learn-about">
+									<a
```

Comment:
You can use the `<ExternalLink />` component from `@wordpress/components` here. It handles the `target` and `rel` attributes for external links automatically and includes the icon, making the code cleaner and more maintainable:

https://wordpress.github.io/gutenberg/?path=/docs/components-externallink--docs

Source code: https://github.com/WordPress/gutenberg/tree/07c7821b07e5d2b2abaa86f6737dd486da26383e/packages/components/src/external-link